pkgs/tools/networking/xdp-tools/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib, stdenv, fetchFromGitHub, libbpf, elfutils, libelf, zlib, libpcap
+, clang, llvm, gnumake, gcc, pkgconfig, m4, emacs-nox, wireshark-cli
+}:
+
+let
+  version = "1.2.5";
+  projectSha256 = "sha256-Kyay5j+87nOZ9C+DI8MI6zhkWqspIHiTfW9Di5uVWzY=";
+
+  src = fetchFromGitHub {
+    owner = "xdp-project";
+    repo = "xdp-tools";
+    rev = "v${version}";
+    sha256 = projectSha256;
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "xdp-tools";
+  inherit version src;
+
+  nativeBuildInputs = [
+    libbpf
+    elfutils
+    libelf
+    zlib
+    libpcap
+    clang
+    llvm
+    gnumake
+    gcc
+    pkgconfig
+    m4
+    emacs-nox
+    wireshark-cli
+  ];
+
+  BPF_CFLAGS = "-fno-stack-protector -Wno-error=unused-command-line-argument";
+  PRODUCTION = 1;
+  DYNAMIC_LIBXDP = 1;
+  FORCE_SYSTEM_LIBBPF = 1;
+  FORCE_EMACS = 1;
+
+  configurePhase = ''
+    ./configure
+  '';
+
+  buildPhase = ''
+    make V=1
+  '';
```

Comment:
Is this necessary?

Reply:
You're right, also `configurePhase` is not necessary